### PR TITLE
Merge knutin/master; fix type declarations

### PR DESCRIPTION
**Pull request by horvand**
We're using the  knutin/edatetime repo as dependency for most of our project, hence the merge.

### CODE REVIEW

File: src/edatetime.erl [hunk 1]
```diff
@@ -18,13 +18,13 @@
 -export([iso8601/1, iso8601_basic/1]).
 -export([tomorrow/1, yesterday/1]).
 
--type timestamp() :: pos_integer().
+-type timestamp() :: integer().
```
Comment (sata) [inline src/edatetime.erl:21]: shouldn't it be non_neg_integer? surely we don't have negative timestamps?

Comment (horvand) [reply src/edatetime.erl:21]: `0` is 1970-01-01, `integer()` is correct as typespec.

Comment (sata) [reply src/edatetime.erl:21]: non_neg_integer() is 0..N while integer() is -1 ... N. Can the timestamp ever be negative? 0 is not negative in this context.

Comment (horvand) [reply src/edatetime.erl:21]: Surely, there must have been something there _before_ the Unix epoch began. :)
I've run into this issue when expressing user's birthdays as timestamps, to give an example.

Comment (sata) [reply src/edatetime.erl:21]: Haha, that's a good edge case :)

**Review by sata — APPROVED** [review]
looks good

**Comment by horvand** [comment]
I'll tag master as `v1.0.2`.

**Pull request merged**